Promise.resolve('abc').then(x => console.log(x));
Promise.reject(new Error(('abc').catch(x => console.error(x));